### DISCUSSION TITLE
Unable to run yarn dev or yarn build on Windows 11

### DISCUSSION THREAD

**Discussion by craph** [requarks/wiki#7276] [Error / Bug Report]
Hello, 

I am trying to build Wiki.js locally on my machine without Docker. And I am unable to run yarn dev or yarn build on my machine.

I have this error : 

```
yarn dev
yarn run v1.22.19
$ NODE_OPTIONS=--openssl-legacy-provider node dev
'NODE_OPTIONS' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
```
If I modify the package.json to add set before NODE_OPTIONS I have this
```
yarn dev
yarn run v1.22.19
$ set NODE_OPTIONS=--openssl-legacy-provider node dev
Done in 0.14s.
```

But nothing more... 

What am I doing wrong ?

Thank you very much.

Best regards,

**Comment by brainauto** [comment]
Setting methods like NODE_OPTIONS=--openssl-legacy-provider are not directly supported on Windows.

follow this 

1. Add cross-env
`yarn add cross-env`

3. modify the 'scripts' section in 'package.json'
default. `scripts: {
    "dev": "NODE_OPTIONS=--openssl-legacy-provider node dev"
}` 
modified. `scripts: {
    "dev": "cross-env NODE_OPTIONS=--openssl-legacy-provider node dev"
}` 

Once you've completed all step you can run yarn dev .
